packages/oui-navbar/src/navbar.html

```diff
@@ -16,7 +16,8 @@
         aria-label="{{::$ctrl.brand.label}}"
         icon-alt="{{::$ctrl.brand.iconAlt}}"
         icon-class="{{::$ctrl.brand.iconClass}}"
-        icon-src="{{::$ctrl.brand.iconSrc}}">
+        icon-src="{{::$ctrl.brand.iconSrc}}"
+        on-click="::$ctrl.brand.onClick">
```

Comment:
I think we should keep the same property name as the other links: `click`. Or else we will have some surprises if we have in one hand `.click` and in the other `.onClick`.

https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-navbar/src/navbar.html#L38

An idea could to support both, but needs to be updated on the others links. What do you think ?

Reply:
yep done